<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iRobotUser</class>
 <widget class="QMainWindow" name="iRobotUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>781</width>
      <height>531</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>781</width>
      <height>531</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QLabel" name="conceptofOperations">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>561</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;iRobot&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>611</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="requestAccount">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>440</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Request Account</string>
      </property>
     </widget>
     <zorder>frame</zorder>
     <zorder>conceptofOperations</zorder>
     <zorder>label</zorder>
     <zorder>requestAccount</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Product Info</string>
     </attribute>
     <widget class="QPushButton" name="requestBrochure">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>450</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Request Brochure</string>
      </property>
     </widget>
     <widget class="QPushButton" name="printBrochure">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>450</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Print Brochure</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>731</width>
        <height>381</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Sales Pitch</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Sales Pitch&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="salesPitch">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>591</width>
          <height>241</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>List of Environments</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>List of Environments</string>
        </property>
       </widget>
       <widget class="QLabel" name="envitemnetList">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>581</width>
          <height>231</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Guarantee Policy</string>
       </attribute>
       <widget class="QLabel" name="guaranteePolicyLabel">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>52</y>
          <width>581</width>
          <height>231</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Guarantee Policy</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Testimonials</string>
     </attribute>
     <widget class="QPushButton" name="addTestimonial">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>470</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Testimonial</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>390</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter your testimonial here</string>
      </property>
     </widget>
     <widget class="QFrame" name="testimonialFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>751</width>
        <height>351</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTextBrowser" name="testimonialDisplay">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>711</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Purchase</string>
     </attribute>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>361</width>
        <height>431</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>291</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="basic">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>211</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Basic Model - $199.99</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="advanced">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>251</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Advanced Robotnics - $249.99</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="hardcore">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>241</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Hardcore Herobot - $299.99</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>340</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tax:</string>
       </property>
      </widget>
      <widget class="QLabel" name="tax">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>340</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>360</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Total:</string>
       </property>
      </widget>
      <widget class="QLabel" name="total">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>360</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>300</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>260</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>260</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string># of robots:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="purchase">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>390</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Purchase</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="maintenanceButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>220</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Maintenance Plan $42,852.60</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="Maintenance">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>150</y>
        <width>241</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="contactus">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>550</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Contact Us</string>
    </property>
   </widget>
   <zorder>contactus</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLog_out"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUser_Guide"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLog_out">
   <property name="text">
    <string>Log out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUser_Guide">
   <property name="text">
    <string>User Guide</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
